app/src/lib/stats/stats-database.ts

```diff
@@ -93,6 +93,9 @@ export interface IDailyMeasures {
 
   /** Whether or not the app has been interacted with during the current reporting window */
   readonly active: boolean
+
+  /** The number of times a merge in Desktop has resulted in conflicts that the user needs to resolve */
+  readonly mergeConflictDetected: number
```

Comment:
spelling: mergeConflict**s**Detected

Reply:
I actually prefer singular "merge conflict" label here to focus this metric on the event rather than changing this to "merge conflicts" which might be interpreted as the number of files conflicted or the number of conflicts in a merge. Thoughts?

Reply:
Or maybe this name can be more explicit: `mergeFailedWithConflictsCount` ?

Reply:
I like the explicit nature of the last one, but what about just `mergeConflictCount`?

Reply:
My gut feeling is still that "conflict" on it's own is open to (mis-)intepretation, and more context here is required to avoid that. 

With a bit more thought I probably would have opened this PR with `mergeConflictDetectedCount` here because we use `Count` for lots of these other metrics, but the bike shed is green, not red :trollface:

Reply:
`countOfMergesWithConflicts`?

Reply:
Or `conflictedMergeCount`?